test/server/http/admin_test.cc

```diff
@@ -13,13 +14,17 @@ class AdminFilterTest : public testing::Test {
 public:
   // TODO(mattklein123): Switch to mocks and do not bind to a real port.
   AdminFilterTest()
-      : admin_("/dev/null", Network::Utility::resolveUrl("tcp://127.0.0.1:9002"), server_),
+      : admin_("/dev/null", "/tmp/envoy.prof", Network::Utility::resolveUrl("tcp://127.0.0.1:9002"),
```

Comment:
This isn't a fault in this PR (it's a widespread issue in Envoy today), but using `/tmp` is kind of dangerous, since you may conflict with other users or test runs in a shared environment. You can keep the code as is, since we don't have a better solution yet, but add a `TODO(htuch): Use proper temporary path allocation method.` here?